docs/AI-for-security/ai-for-security.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[ai-for-security]]
+= AI for security
+
+:frontmatter-description: Learn to use AI capabilities in {elastic-sec}.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [overview]
+:frontmatter-tags-user-goals: [get-started]
+
+You can use {elastic-sec}'s built-in AI tools to speed up your work and augment your team's capabilities. The pages in this section describe <<security-assistant, AI Assistant>>, which answers questions and enhances your workflows throughout {elastic-sec}, and <<attack-discovery, Attack discovery>>, which speeds up the triage process by finding patterns and identifying attacks spanning multiple alerts.
+
+include::security-assistant.asciidoc[leveloffset=+1]
+include::attack-discovery.asciidoc[leveloffset=+1]
+
+include::ai-use-cases.asciidoc[leveloffset=+1]
+include::ai-alert-triage.asciidoc[leveloffset=+2]
+include::use-attack-discovery-ai-assistant-incident-reporting.asciidoc[leveloffset=+2]
+include::ai-esql-queries.asciidoc[leveloffset=+2]
+
+include::llm-connector-guides.asciidoc[leveloffset=+1]
+include::azure-openai-setup.asciidoc[leveloffset=+2]
+include::connect-to-openai.asciidoc[leveloffset=+2]
+include::connect-to-bedrock.asciidoc[leveloffset=+2]
+
```

Comment:
This is a nit, but for consistency, it would be nice to have the child pages under the set up connectors section
 listed in the same order in the left nav and on the page itself. Same goes for the use cases section. 
 
![image](https://github.com/elastic/security-docs/assets/137783811/00664f32-cb2f-4b71-a863-e5be94ffe02f)